cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc

```diff
@@ -69,7 +69,7 @@ Here we detail the available versions:
 | Query option | Description | Default
 | `3.5` | This will force the query to use Neo4j Cypher 3.5. |
 | `4.1` | This will force the query to use Neo4j Cypher 4.1. |
-| `4.2` | This will force the query to use Neo4j Cypher 4.2. As this is the default version, it is not necessary to use this option explicitly. | X
+| `4.2` | This will force the query to use Neo4j Cypher 4.2. As this is the default version, it is not necessary to use this option explicitly. | {check-mark}
```

Comment:
```
//Check Mark
:check-mark: icon:check[]

//Cross Mark
:cross-mark: icon:times[]
```

Add this at line 3, see https://github.com/neo4j/neo4j-documentation/commit/1337e41225a2c9c92b6721b103144fb1dc0f3a37